### DISCUSSION TITLE
how to reset repeat_interval when alerts get resolved ?

### DISCUSSION THREAD

**Discussion by yizhixiaotaozi** [prometheus/alertmanager#3524] [Q&A]
Suppose there is an alert fired at 00:00 and is resolved at 00:05, and repeat_interval is 2h。
If this alert is refired at 00:15, alertmanager will not send webhook notification until 02:00。

Am I understanding correctly ?  I hope repeat_interval is recalculated when this group is resolved，is that possible?

**Comment by grobinson-grafana** [comment]
That doesn't sound correct to me, it will send a webhook notification at the next Group Interval, not the next Repeat Interval, as the alert resolved and then fired again.
